web/src/routes/profile.tsx

```diff
@@ -0,0 +1,25 @@
+import {useAuth} from '@/context/auth-provider';
+import {Card} from '@/components/ui/card';
+import {createFileRoute} from '@tanstack/react-router';
+
+export const Route = createFileRoute('/profile')({
+  component: RouteComponent,
+});
+
+/**
+ * RouteComponent component renders the profile page.
+ * It displays a card with the user's profile information.
+ *
+ * @returns {JSX.Element} The rendered RouteComponent component.
+ */
+function RouteComponent() {
+  const auth = useAuth();
+
```

Comment:
Not sure we need both 'Account' and 'Profile' but you could usefully list the global roles that the user has here (cluster-admin, notebook-creator).  Maybe also what auth method they have used (local password, google).